_sass/leaderboard.scss

```diff
@@ -29,6 +29,31 @@
   }
 }
 
+.leaderboard.dark {
+  
+  a {
+    color: white;
+  }
+
+  tr:nth-child(2n + 1) {
+    color: white;
+  }
+
+  tr:nth-child(2n) {
+    background-color: grey;
+    color: white;
```

Comment:
And then we can remove this line since it's covered in our base case for all `tr`'s

```suggestion
```